kitchen-sink/src/views/MainPanel.js

```diff
@@ -32,27 +30,24 @@ const MainPanel = kind({
 			<Row align="center space-around">
 				<Cell shrink>
 					<Button>Click me</Button>
-					<Button type="standard" icon="home" />
-					<ToggleButton
-						underline
-						type="standard"
-						toggleOffLabel="Off"
-						toggleOnLabel="On"
-					/>
+					<Button icon="plus" type="standard">Home</Button>
```

Comment:
Wasn't this supposed to be an icon button? I mean just a button with a "home" icon.

Reply:
Done